docs/development/plugins/embeddable/public/kibana-plugin-plugins-embeddable-public.embeddableinput.md

```diff
@@ -19,5 +19,6 @@ export declare type EmbeddableInput = {
     timeRange?: TimeRange;
     query?: Query;
     filters?: Filter[];
+    searchSessionId?: string;
```

Comment:
This is a major nit, but do we really want to have the name be so specific?
I think sessions can and will be used for more than search.

Reply:
I think `sessionId` is way too vague for dashboards / embeddable level, but OK on search level. 
`sessionId` on dashboard could mean anything: 
* user authentication session 
* dashboard page session (created once user when user loaded a dashboard) 
* dashboard app session (to track drill-downs between dashboards without navigating away from a dashboard app)
* and the search / fetch session.

Reply:
Agreed, if this terminology is for using in solutions :)